src/scripts/createMarkdown.sh

```diff
@@ -57,8 +57,80 @@ if [ -f "$FILE_PATH" ]; then
   exit 1
 fi
 
-# Define additional section for native files
-NATIVE_ADDITIONAL_SECTIONS=$(cat <<'EOF'
+# Define section for native files
+NATIVE_SECTIONS=$(cat <<'EOF'
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a button
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Skip-links: Focus moves directly to the header or navigation
+
+   - Tab: Nothing, headings are not focusable unless they are actionable
+
+   - Arrow-keys: Headings are browsed
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves directly to the header or navigation
+
+   - Doubletap: This typically activates most elements
+
+3. Listen to screenreader output on all devices
+
+   - It is discoverable with screenreader shortcuts as header/banner landmark
+
+   - Group: It typically contains the name and primary navigation of the website
+
+4. Test device settings
+
+   - Text resize: Text can resize up to 200% without losing information
+
+Full information: [$FULL_LINK]($FULL_LINK)
+
+## Gherkin
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a button
+
+GIVEN THAT I am on a screen with a button
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
+
+   - I use the tab key to enter the web browser window
+
+   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+
+   - I HEAR It typically contains the name and primary navigation of the website
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+
+   - I swipe to focusable elements in the header
+
+   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
+
+   - I HEAR It typically contains the name and primary navigation of the website
+
+4. Scenario: Test device OS settings for text resize
+
+   - WHEN I adjust the device text resize setting to 200%
+   
+      - THEN the text on the button should resize up to 200% without losing information
+
+Full information: [$FULL_LINK]($FULL_LINK)
```

Comment:
The full link doesn't seem to be populating correctly for the template, it shows up just as it is shown here in line 133.

Reply:
great catch! fixed!